### PR TITLE
🚀  Release v1.8.0-beta.1

### PR DESCRIPTION
**Pull request by vishalanarase**
/area release

### CODE REVIEW

File: CHANGELOG/v1.8.0-beta.1.md [hunk 1]
```diff
@@ -0,0 +1,419 @@
...
+- KCP/ClusterResourceSet: Do not update observed generation if there are reconcile errors (#10736)
+- KCP/MachineSet: Flag for old infra machine naming (#10576)
+- Machine/MachineSet: Remove redundant watch event handlers via `Owns()` (#10048)
+- MachineSet/MachineDeployment: Preserve finalizers during MS/Machine reconciliation (#10694)
+- Release: Prepare main branch for v1.8 development (#10524)
+- Release: Push manifests for main & release-.* (#10521)
+- Release: Release notes: also detect alpha releases as pre releases (#10369)
+- Release: Release v1.5.8 (#10438)
+- Release: Release v1.6.4 (#10439)
+- Release: Release v1.7.0 (#10440)
```
[the remark this answers is not in the document]
Comment (vishalanarase) [reply CHANGELOG/v1.8.0-beta.1.md]: Done

File: CHANGELOG/v1.8.0-beta.1.md [hunk 2]
```diff
@@ -0,0 +1,416 @@
+🚨 This is a BETA RELEASE. Use it only for testing purposes. If you find any bugs, file an [issue](https://github.com/kubernetes-sigs/cluster-api/issues/new).
+
+## Highlights
+* Add kubeadm v1beta4 types
```
Comment (sbueringer) [inline CHANGELOG/v1.8.0-beta.1.md:4]: Just fyi. I will provide a highlights and deprecation section. Not sure if already for the beta tomorrow, but at least for the next releases

File: CHANGELOG/v1.8.0-beta.1.md [hunk 3]
```diff
@@ -0,0 +1,416 @@
...
+
+:warning: **BETA RELEASE NOTES** :warning:
+## 👌 Kubernetes version support
+
+- Management Cluster: v1.27.x -> v1.30.x
+- Workload Cluster: v1.25.x -> v1.30.x
+
+[More information about version support can be found here](https://cluster-api.sigs.k8s.io/reference/versions.html)
+
+## Changes since v1.7.0
```
Comment (sbueringer) [inline CHANGELOG/v1.8.0-beta.1.md:18]: A few more (just merged):
* #10912
* #10849 (merging)

**Review by sbueringer — COMMENTED** [review]
Otherwise lgtm from my side

File: CHANGELOG/v1.8.0-beta.1.md (hunk 3 above)
Comment (vishalanarase) [reply CHANGELOG/v1.8.0-beta.1.md:18]: PR #10912 and  #10849 added, #10922 isn't because it wasn't merged

Comment (vishalanarase) [reply CHANGELOG/v1.8.0-beta.1.md:18]: Added now

File: CHANGELOG/v1.8.0-beta.1.md [hunk 4]
```diff
@@ -0,0 +1,418 @@
...
+- Dependency: Bump to Go 1.22.5 (#10828)
+- Dependency: Fix loopvar linter issue and usage of deprecated grpc function (#10731)
+- Dependency: Fixup patch order in CABPK (#10399)
+- Dependency: Stop bumping cel-go via dependabot (#10834)
+- Dependency: Update Kustomize deprecated syntax (#10294)
+- Devtools: Add triage-party for the Cluster API backlog (#10437)
+- Devtools: Bump Cluster API Visualizer to v1.3.0 (#10386)
+- Devtools: Bump Cluster API Visualizer to v1.3.1 (#10816)
+- Devtools: Improve triage query (#10644)
+- Devtools: Tilt ensure in-tree providers always use start.sh to allow restarts (#10811)
```
Comment (sbueringer) [inline CHANGELOG/v1.8.0-beta.1.md:146]: Honestly, dropping all those colons just makes it had to read. Can we please keep them?

File: CHANGELOG/v1.8.0-beta.1.md (hunk 3 above)
Comment (sbueringer) [reply CHANGELOG/v1.8.0-beta.1.md:18]: @vishalanarase I don't see 10922

Comment (sbueringer) [reply CHANGELOG/v1.8.0-beta.1.md:18]: Please also add 10918

File: CHANGELOG/v1.8.0-beta.1.md (hunk 4 above)
Comment (vishalanarase) [reply CHANGELOG/v1.8.0-beta.1.md]: Addressed

File: CHANGELOG/v1.8.0-beta.1.md (hunk 3 above)
Comment (vishalanarase) [reply CHANGELOG/v1.8.0-beta.1.md:18]: Added

**Review by chandankumar4 — APPROVED** [review]
looks good to me. Thanks

**Review by Sunnatillo — COMMENTED** [review]
LGTM

**Comment by adilGhaffarDev** [comment]
checked the last commits, they are in the notes. I am approving it. @sbueringer comments are also addressed, can we unhold this? 
/approve

File: CHANGELOG/v1.8.0-beta.1.md (hunk 3 above)
Comment (sbueringer) [reply CHANGELOG/v1.8.0-beta.1.md:18]: Thx!

File: CHANGELOG/v1.8.0-beta.1.md (hunk 4 above)
Comment (sbueringer) [reply CHANGELOG/v1.8.0-beta.1.md]: Thank you!

**Comment by sbueringer** [comment]
/lgtm

all good from my side

**Pull request merged**